cmd/optimize/report.go

```diff
@@ -128,6 +128,11 @@ func listReports(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("extractReportData: %w", err)
 	}
 
+	if len(reportRows) < 1 {
+		cmd.Println("No results found for given input")
```

Comment:
Can you please make this conditional on table output format only? I.e., not to show this if -o json or yaml. I am not sure where cmd.Println goes (stdout/err); I think we prefer output.PrintCommandStatus (with a \n) but that's not hard & fast.

Reply:
Done